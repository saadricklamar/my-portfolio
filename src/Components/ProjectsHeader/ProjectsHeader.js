import React from "react";
import "./ProjectsHeader.scss";

function ProjectsHeader(props) {
  return (
    <header className="projects-header">
      <button
        className="project-buttons"
        id={`${props.allButton}`}
        onClick={props.renderAll}
      >
        All
      </button>
      <button className="project-buttons" onClick={props.renderReact}>
        React
      </button>
      <button className="project-buttons" onClick={props.renderVue}>
        Vue.js
      </button>
      <button className="project-buttons" onClick={props.renderNode}>
        Node.js
      </button>
    </header>
  );
}

export default ProjectsHeader;
