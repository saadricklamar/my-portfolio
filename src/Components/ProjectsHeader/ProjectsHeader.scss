.projects-header {
  display: flex;
  justify-content: center;
  margin: 5% auto 0 auto;
}

.project-buttons {
  background: none;
  border: none;
  font-family: "Titillium Web", sans-serif;
  font-size: 2rem;
  height: 50px;
  margin: 8% 1% 5% 0;
  outline: none;
  width: 170px;

  &:hover {
    background: #f82a7b;
    border-radius: 2px;
    color: white;
    cursor: pointer;
    transition: 1s ease;
  }

  &:focus {
    background: #f82a7b;
    border-radius: 2px;
    color: white;
    cursor: pointer;
  }
}

@media (max-width: 800px) {
  .project-buttons {
    width: 120px;
    margin: 15% 1% 5% 0;
  }
}

@media (max-width: 420px) {
  .project-buttons {
    width: 90px;
    font-size: 1rem;
    margin: 15% 0 5% 0;
  }
}
