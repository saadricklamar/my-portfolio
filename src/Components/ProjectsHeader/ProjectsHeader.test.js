import React from "react";
import { shallow } from "enzyme";
import ProjectsHeader from "./ProjectsHeader";

describe("ProjectsHeader", () => {
  let wrapper = shallow(<ProjectsHeader />);
  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
