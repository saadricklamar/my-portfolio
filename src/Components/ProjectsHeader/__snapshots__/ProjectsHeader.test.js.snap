// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsHeader should match the snapshot 1`] = `ShallowWrapper {}`;
